### ISSUE TITLE
Submit UI: too difficult to choose a topic

### ISSUE THREAD

**Issue by hugoroy** [tosdr/tosdr2#24]
Right now, you select Topics like you select Services. However, these are very different.

For a service, you don't have to think, you just have to write the name of the service, so the completion works fine.

For a topic, you need to think and to choose, among the many topics. So first, you need to have a good overview of what you can choose from. In the current UI,  the submitter is left entirely clueless (unless you already know all the topics and which topics you'll assign — but that's only very, very experienced ToS;DR contributors).

We need to rethink the UI for this. @jancborchardt do you have ideas? 

I think that a 3-column list of all topics where you can check the boxes would work fine. Also try to order the topics in some way. IIRC, some topics are broader "categories" than others.

**Comment by pierreozoux** [comment]
Actually, when you click on the field, you get the full list as a dropdown.
We could add a `?` next to it, point to https://edit.tosdr.org/topics a bit like this:

[?](https://edit.tosdr.org/topics)

What do you think?

**Comment by hugoroy** [comment]
Yes, but the dropdown isn't great for topics. For services, it works fine.

I don't think the "?" would cut it. 

I think something like the "index of topics" shown here https://tosdr.org/topics.html would work, provided that the topics are not links, but have a little checkbox on the left. Something like that where you can see at a glance all topics.

**Comment by hugoroy** [comment]
Give the possibility for users to ignore the topics if they don't know which one to choose. We can always let the topic be figured out later for users who are more experienced.

**Comment by michielbdejong** [comment]
Also, in order to rate a point within a topic, it is necessary to compare it to similar points about other services. This is what I have been trying to set up with the 'cases'.

@piks3l, can we find some time to brainstorm about this?
